tests/results/result_actions/ApplyPatchActionTest.py

```diff
@@ -115,7 +115,8 @@ def test_apply_rename(self):
                       file_diff_dict)
             self.assertFalse(isfile(f_a+'.renamed.orig'))
 
-            file_dict = {f_a+'.renamed': open(f_a+'.renamed').readlines()}
+            with open(f_a+'.renamed') as fh:
```

Comment:
This needs to be fixed in this PR, since otherwise AFAIU the tests would fail due to the new update of pytest to version 3.1. Why do you think that this fix is different from the other ones? (I also fixed a ResourceWarning in Beartest.py)
```
tests/results/result_actions/ApplyPatchActionTest.py::ApplyPatchActionTest::test_apply_rename
  /home/thomas/gitrepos/coala/tests/results/result_actions/ApplyPatchActionTest.py:118: ResourceWarning: unclosed file <_io.TextIOWrapper name='/tmp/tmpbu9i6owm.renamed' mode='r' encoding='UTF-8'>
    file_dict = {f_a+'.renamed': open(f_a+'.renamed').readlines()}
```